stable/ambassador/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 0.51.2
+appVersion: 0.52.0
```

Comment:
update `image.tag` in values as well :) thats the important one. I know there used to be a template which selected the app version if no image tag was set but I felt this was a bit to magic.

Reply:
Whoops. Done, and in the README too.